### ISSUE TITLE
ICEE SOW

### ISSUE THREAD

**Issue by neu6** [ICPI/Coordination#4]
**ICPI Capacity Building, Enrichment, and Engagement (ICEE) Team 
Statement of Work**

The ICPI Capacity Building, Enrichment, and Engagement (ICEE) Team is an interdisciplinary team working to manage ICPI engagement requests (HQ and Field) and capacity building learning strategies. The primary objectives of the ICE Team are summarized below:

•	Engage with HQ, Field, ECTs, and agencies to encourage regular communication with ICPI, especially for analytic support requests, training needs, TDY support, etc. Maintain open channels of communication with various audiences and ensure they are aware of ICPI’s support-related ability and availability.  
•	Manage and interpret the processes around ICPI engagements
        o	Manage the ICPI Engagement requests including the processes around the Engagement Form submitted by the requestor and the Engagement Log once ICPI is actively involved in an engagement
        o	Liaise between the requestor and ICPI team to enhance communication when needed
        o	Communicate updates, deadlines, next steps, and other relevant information to all relevant parties involved in the ICPI Engagement
•	Create efficiencies and transparency in the engagement processes by
        o	Following-up on the engagement next steps and timelines
        o	Enhancing information exchange and knowledge sharing
        o	Documenting meaningful deliverables and outputs from engagement 
•	Help develop, implement, and sustain various capacity building learning strategies at ICPI
        o	Provide internal ICPI guidance and strategic direction in collaboration with the DIV and OCM Teams to groups that focus on specific elements of capacity building
       o	Maintain a bridge between ICPI and PRIME to ensure learning strategies are in sync
        o	Coordinate within ICPI and other teams at S/GAC that are developing learning portals, tools, materials, etc. 
•	Maintain a link to the country teams (via a Field Advisor Board-type entity) to improve capacity building tools, brainstorm innovative analyses ideas, and generally provide a country-level perspective to ICPI’s work

**ICEE Team Membership**
A member from each team within the Coordination Cluster will be asked to join the ICEE Team (on an as needed basis in the beginning) to maintain an interdisciplinary and interagency effort in furthering ICPI’s engagement and capacity building goals.

**Comment by shapaklyak** [comment]
@kkale25 agreed...like the grid format too! will be good to continue talking through this :)

**Comment by jb3436** [comment]
@kkale25 reviving this
Let's distribute at the next staff meeting for review and post it on SharePoint and in the wiki pages under resources

FYI @kkale25 @hessionra
